administrator/modules/mod_menu/tmpl/default_submenu.php

```diff
@@ -124,6 +124,13 @@
 
 $itemImage = (empty($itemIconClass) && $itemImage) ? '&nbsp;<img src="' . Uri::root() . $itemImage . '" alt="">&nbsp;' : '';
 
+// if the item image is not set, than the item title would not have margin
+// here we add it
```

Comment:
```suggestion
// If the item image is not set, the item title would not have margin. Here we add it.
```
The "than" was wrong English, it should have been "then". But it's better to remove it completely here.

We should avoid multi-line comments, too. (But if we use them they should have different code style, `//`we should use only for single line comments).

Reply:
@eopws Please change that comment as suggested. It will not require new human tests after the change.